common_arm/usb_cdc.c

```diff
@@ -649,7 +654,7 @@ bool usb_poll(void) {
 bool usb_poll_validate_length(void) {
     if (!usb_check()) return false;
     if (!(pUdp->UDP_CSR[AT91C_EP_OUT] & btReceiveBank)) return false;
-    return ((pUdp->UDP_CSR[AT91C_EP_OUT] & AT91C_UDP_RXBYTECNT) >> 16) >  0;
+    return usb_available_length() > 0;
```

Comment:
I suspect this call makes things slower...

Reply:
Yeah that's true. The gcc tells me the inlining failed there.